mmv1/products/datastream/Stream.yaml

```diff
@@ -764,6 +784,211 @@ properties:
+                      description: |
+                        Schema name.
+                    - !ruby/object:Api::Type::Array
+                      name: 'tables'
+                      min_size: 1
+                      description: |
+                        Tables in the database.
+                      item_type: !ruby/object:Api::Type::NestedObject
+                        description: |
+                          SQL Server table.
+                        properties:
+                          - !ruby/object:Api::Type::String
+                            name: 'table'
+                            required: true
+                            description: |
+                              Table name.
+                          - !ruby/object:Api::Type::Array
+                            name: 'columns'
+                            min_size: 1
+                            description: |
+                              SQL Server columns in the schema. When unspecified as part of include/exclude objects, includes/excludes everything.
+                            item_type: !ruby/object:Api::Type::NestedObject
+                              description: |
+                                SQL Server Column.
+                              properties:
+                                - !ruby/object:Api::Type::String
+                                  name: 'column'
+                                  description: |
+                                    Column name.
+                                - !ruby/object:Api::Type::String
+                                  name: 'dataType'
+                                  description: |
+                                    The SQL Server data type. Full data types list can be found here:
+                                    https://learn.microsoft.com/en-us/sql/t-sql/data-types/data-types-transact-sql?view=sql-server-ver16
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'length'
+                                  output: true
+                                  description: |
+                                    Column length.
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'precision'
+                                  output: true
+                                  description: |
+                                    Column precision.
+                                - !ruby/object:Api::Type::Integer
+                                  name: 'scale'
+                                  output: true
+                                  description: |
+                                    Column scale.
+                                - !ruby/object:Api::Type::String
+                                  name: 'encoding'
+                                  output: true
+                                  description: |
+                                    Column encoding.
```

Comment:
I don't see this in the documentation: https://cloud.google.com/datastream/docs/reference/rest/v1/projects.locations.streams#SqlServerColumn

```suggestion
```